docs/pages/server-access/rbac.mdx

```diff
@@ -0,0 +1,183 @@
+this section focuses on configuring RBAC for Server Access.
+
+## Role configuration
+
+Teleport's "role" resource provides the following instruments for restricting
+server access:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: developer
+spec:
+  allow:
+    # The logins array defines the OS/UNIX logins a user is allowed to use.
+    # both strings and template variables are supported in this field
+    logins: [ubuntu, debian, '{{internal.logins}}']
+
+    # node_labels: a user with this role will be allowed to connect to
+    # SSH nodes, which labels match expressions below.
+    node_labels:
+      # literal strings:
+      'env': 'test'
+      # the wildcard ('*') means "any node"
+      '*': '*'
+      # a list of alternative options:
+      'region': ['us-west-1', 'eu-central-1']
+      # regular expressions start with ^ and end with $
+      # Teleport uses golang regexp syntax.
+      # of the list example above can be expressed as:
+      'reg': '^us-west-1|eu-central-1$'
+
+    # List of host groups the created user will be added to. Any that don't already exist are created.
+    host_groups: [ubuntu, nginx, other]
+
+    # Assign the user to the sudoers group
+    host_sudoers:
+    - 'ALL=(ALL) NOPASSWD: ALL'
+
+    # Deny access to the root user
+  deny:
+    logins:
+    - root
+
+  options:
+   # Automatically create the user if does not exist
+   # The user is deleted after the session ends if Teleport created the user. 
+    create_host_user: true
+```
+
+
+<Admonition
+  type="note"
+  title="Deny Rules"
+>
+  Deny rules will match greedily. In the example above, a server session
+  attempting to use "root" serveruser  account will be rejected.
+</Admonition>
+
+## Template variables
+
+Similar to other role fields, the Server related fields support templating variables.
+
+The `{{external.xyz}}` variables are replaced with values from external [SSO](../access-controls/sso.mdx)
+providers. For OIDC, they will be expanded with a value of an "xyz" claim; for
+SAML — with an "xyz" assertion value.
+
+For example, here is what a role may look like if you want to assign allowed
+server environment types from the user's Okta `environments` assertion:
+
+```yaml
+spec:
+  allow:
+    node_labels:
+    - env:  '{{external.environments}}'
+```
+
+The `{{internal.logins}}` variable permits sharing
+allowed server logins with remote clusters. They will be replaced
+with the respective properties of a remote user connecting from a root cluster.
+
+For example, suppose a user in the root cluster has the following role:
+
+```yaml
+spec:
+  allow:
+    logins: ["ubuntu"]
+```
+
+The role on the leaf cluster can be set up to use the user's allowed server
+accounts and names:
+
+```yaml
+spec:
+  allow:
+    logins: ["{{internal.logins}}"]
+```
+
+## Server role options
+
+The following options in a Teleport role apply to Server Access.
+
+```yaml
```

Comment:
Would this be a good candidate for a partial? I assume there's very similar content on the main RBAC page, right?

Reply:
As there are differences in things like the recorded sessions area I'd say no.  Otherwise we'd have to have multiple partial splits.